pulsar/resource_pulsar_tenant_test.go

```diff
@@ -173,7 +173,8 @@ provider "pulsar" {
 }
 
 resource "pulsar_tenant" "test" {
-  tenant = "thanos"
+	tenant = "thanos"
+	allowed_clusters = ["standalone"]
```

Comment:
Note: this change was to fix the following error that surfaced when moving the integration tests from Pulsar 2.4.1 to 2.7.0

```
=== RUN   TestTenant
    testing.go:635: Step 0 error: errors during apply:
        
        Error: ERROR_CREATE_TENANT: code: 412 reason: Clusters can not be empty
         request_input: utils.TenantData{Name:"thanos", AdminRoles:[]string{}, AllowedClusters:[]string{}}
        
          on /var/folders/f2/7c9_sq9j46sb4nbywp3w1dbxr8xj5b/T/tf-test095334148/main.tf line 6:
          (source code not available)
```